config/helm/chart/default/templates/Openshift/oneagent/securitycontextconstraints-privileged.yaml

```diff
@@ -0,0 +1,66 @@
+{{- $platformIsSet := printf "%s" (required "Platform needs to be set to kubernetes, openshift " (include "dynatrace-operator.platformSet" .))}}
+{{- if and (eq .Values.platform "openshift") ((.Values.securityContextConstraints).enabled) (eq (include "dynatrace-operator.partial" .) "false")}}
+# Copyright 2021 Dynatrace LLC
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+apiVersion: security.openshift.io/v1
+kind: SecurityContextConstraints
+metadata:
+  annotations:
+    kubernetes.io/description: "dynatrace-dynakube-oneagent-privileged allows access to all privileged and host features and the ability to run as any user, any group, any fsGroup, and with any SELinux context."
+  name: dynatrace-dynakube-oneagent-privileged
+  labels:
+    {{- include "dynatrace-operator.oneagentLabels" . | nindent 4 }}
+allowHostDirVolumePlugin: true
+allowHostIPC: false
```

Comment:
```suggestion
```
Nit: isn't it `false` by default?

Reply:
Either assign all of fields with default values or none of them. Selectively choosing fields is confusing.

Reply:
privileged SCC is a copy of the unprivileged SCC, no other changes were made